python/whylogs/core/resolvers.py

```diff
@@ -110,17 +118,21 @@ class ResolverSpec:
     """
 
     column_name: Optional[str] = None  # TODO: maybe make this a regex
-    column_type: Optional[DataType] = None
+    column_type: Optional[Union[DataType, Type]] = None
```

Comment:
```suggestion
    column_type: Type = None
```

Reply:
Actually, `DataType` would imply an instance of the `DataType` class, but we mean a subclass of `DataType`. I think that's problematic across the range of Python versions we support, so perhaps `Type` after all...